runtime/lua/vim/lsp.lua

```diff
@@ -870,9 +870,12 @@ function lsp.omnifunc(findstart, base)
   local textMatch = vim.fn.match(line_to_cursor, '\\k*$')
   local params = util.make_position_params()
 
+  local items = {}
```

Comment:
@justinmk one modification, `complete()` isn't exactly clear, but for multiple servers, I think this would override the complete menu, so I maintain my own copy of all the results and set the concatenation every time. I don't have something with multiple servers to test it, though...

I tried complete_add for this, but I wasn't having success.

Reply:
I'm not sure either. Let's update the doc for complete() if we find out...